study.md

```diff
@@ -40,14 +41,19 @@ What does an exclamation point after a method usually signify?  e.g.,
 `my_string.capitalize!`
 
 ```md
-<!-- your answer here -->
+It says that I want to use again the array that I have created before
 ```
 
 ## Instantiation
 How do you create an instance of a class in Ruby?
 
 ```ruby
-# your answer here
+class stone
```

Comment:
This defines a class but doesn't instantiate an instance of it.  `class_instance = ClassName.new`.